lib/strict.t

```diff
@@ -18,8 +18,14 @@ like($@, qr/^Unknown 'strict' tag\(s\) 'foo bar'/);
 eval qq(no strict qw(foo bar));
 like($@, qr/^Unknown 'strict' tag\(s\) 'foo bar'/);
 
-eval 'use v5.12; use v5.10; ${"c"}';
-is($@, '', 'use v5.10 disables implicit strict refs');
+{
+    my $warnings = "";
+    local $SIG{__WARN__} = sub { $warnings .= $_[0] };
+    eval 'use v5.12; use v5.10; ${"c"}';
+    is($@, '', 'use v5.10 disables implicit strict refs');
+    ok($warnings =~ m/^Downgrading a use VERSION declaration to below v5.11 is now deprecated at /,
+        'use 5.01 after use 5.012 provokes deprecation warning');
```

Comment:
The use of different version number formats (especially the truncated one) in the test description from the actual test is confusing.

Reply:
Oops I think that may be a copy-paste error. I'll fix it up